apps/bc-wallet-api-server/src/database/schema/tenantsToUsers.ts

```diff
@@ -0,0 +1,33 @@
+import { index, pgTable, primaryKey, text, timestamp, uuid } from 'drizzle-orm/pg-core'
+import { relations } from 'drizzle-orm'
+import { tenants } from './tenants'
+import { users } from './user'
+
+export const tenantsToUsers = pgTable(
+  'tenantsToUsers',
+  {
+    tenant: text()
+      .references(() => tenants.id, { onDelete: 'cascade' })
+      .notNull(),
+    user: uuid()
+      .references(() => users.id, { onDelete: 'cascade' })
+      .notNull(),
+    createdAt: timestamp('created_at').defaultNow().notNull(),
+    updatedAt: timestamp('updated_at')
+      .defaultNow()
+      .notNull()
+      .$onUpdate(() => new Date()),
+  },
+  (t) => [primaryKey({ columns: [t.tenant, t.user] }), index('tenants_to_users_tenant_idx').on(t.tenant)],
+)
```

Comment:
maybe also add
```suggestion
  index('tenants_to_users_user_idx').on(t.user)
 )
```
? If we want to fetch the tenant(s) belonging to a user

Reply:
Index added